### PR TITLE
Update Homeassistant to 2023.6

### PR DESCRIPTION
**Pull request by Xosten**
Bumping Homeassistant from 2023.2 to 2023.6. 

Full Changelog of 2023.6 is available here: https://www.home-assistant.io/changelogs/core-2023.6

### CODE REVIEW

File: home-assistant/docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_ADD: "false"
 
   server:
-    image: homeassistant/home-assistant:2023.2.0@sha256:a4a9f614a1edcd414d63decd99813a2ff9910d876db1e04641540a0c75d20918
+    image: homeassistant/home-assistant:2023.6@sha256:d1dadb8e6ae23c76875c24b72c7af85cbf0e35fd0f6fbfdb27f6ec56741bb7b9
```
Comment (nmfretz) [inline home-assistant/docker-compose.yml:11]:
```suggestion
    image: homeassistant/home-assistant:2023.6@sha256:8e069fcfddbd95c0acff88b9e515e3b208c2a5b4ad801a8f0cb8b55be417fc45
```

**Review by nmfretz — COMMENTED** [review]
Thanks for updating Home Assistant @Xosten! Looks like the wrong sha256 digest was included. I suggested the correct one. Once that's committed I'll test this out and merge.

**Comment by Xosten** [comment]
> Thanks for updating Home Assistant @Xosten! Looks like the wrong sha256 digest was included. I suggested the correct one. Once that's committed I'll test this out and merge.

Thanks Nathan! I’m sorry, couldn’t test the update as I don’t have currently access to my test environment 😕.

**Comment by nmfretz** [comment]
Tested on amd64 and arm64. Thanks again @Xosten!

**Pull request merged**